tracking/src/main/java/org/hps/recon/tracking/kalman/KalTrack.java

```diff
@@ -29,14 +29,16 @@
     public double alpha;
     private double[][] Cx;
     private double[][] Cp;
-    public double Bmag;
+    double Bmag;
     private Vec tB;
     private double time;
+    private boolean verbose;
     double tMin;
     double tMax;
 
     KalTrack(int evtNumb, int tkID, int nHits, ArrayList<MeasurementSite> SiteList, double chi2) {
         // System.out.format("KalTrack constructor chi2=%10.6f\n", chi2);
+        verbose = false;
```

Comment:
Very minor: Instead of hard-coded verbose level, you could consider making use of a logger, which default will be not-verbose, but can be turned on with a properties file.
This is entirely up to you, I just want you to be aware this option exists.